package.json

```diff
@@ -10,6 +10,9 @@
     "configure-vxdev-env": "node libs/utils/build/scripts/generate_env_file.js --isVxDev -o /vx/config/.env.local",
     "run-dev": "script/run-dev"
   },
+  "engines": {
+    "node": "20.16.0"
```

Comment:
Crazy idea but how trivial would it be to upgrade to node v22.x?

Reply:
I wish - sadly, it's a can of worms. Tried upgrading to v22 back when we went from v16 to v20, but there's a breaking change to the native node-api that affects a lot of our native  dependencies.

Technically, VxDesign should only need `node-canvas`, as far as I'm aware (which I think has a newer 22-compatible version), so we could've theoretically gotten away with v22 just for this deployment if we had an easy way to pnpm install and build only the pieces needed for VxDesign